src/executor/admin/ShowStatusExecutor.cpp

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+*
+* This source code is licensed under Apache 2.0 License,
+* attached with Common Clause Condition 1.0, found in the LICENSES directory.
+*/
+
+#include "executor/admin/ShowStatusExecutor.h"
+#include "context/QueryContext.h"
+#include "service/PermissionManager.h"
+#include "planner/Admin.h"
+#include "util/SchemaUtil.h"
+#include "util/ScopedTimer.h"
+
+namespace nebula {
+namespace graph {
+
+folly::Future<Status> ShowStatusExecutor::execute() {
+    SCOPED_TIMER(&execTime_);
+
+    auto spaceId = qctx()->rctx()->session()->space().id;
+    return qctx()->getMetaClient()->getStatis(spaceId).via(runner()).then(
+        [this, spaceId](StatusOr<meta::cpp2::StatisItem> resp) {
+            if (!resp.ok()) {
+                LOG(ERROR) << "SpaceId: " << spaceId
+                           << ", Show staus failed: " << resp.status();
+                return resp.status();
+            }
+            auto statisItem = std::move(resp).value();
+
+            DataSet dataSet({"Type", "Name", "Count"});
+            std::vector<std::pair<std::string, int64_t>> tagCount;
+            std::vector<std::pair<std::string, int64_t>> edgeCount;
+
+            for (auto &tag : statisItem.get_tag_vertices()) {
+                tagCount.emplace_back(std::make_pair(tag.first, tag.second));
+            }
+            for (auto &edge : statisItem.get_edges()) {
+                edgeCount.emplace_back(std::make_pair(edge.first, edge.second));
+            }
+
+            std::sort(tagCount.begin(), tagCount.end(), [](const auto& l, const auto& r) {
+                return l.first < r.first;
+            });
+            std::sort(edgeCount.begin(), edgeCount.end(), [](const auto& l, const auto& r) {
+                return l.first < r.first;
+            });
+
+            for (auto &tagElem : tagCount) {
+                Row row;
+                row.values.emplace_back("Tag");
+                row.values.emplace_back(tagElem.first);
+                row.values.emplace_back(tagElem.second);
+                dataSet.rows.emplace_back(std::move(row));
+            }
+
+            for (auto &edgeElem : edgeCount) {
+                Row row;
+                row.values.emplace_back("Edge");
+                row.values.emplace_back(edgeElem.first);
+                row.values.emplace_back(edgeElem.second);
+                dataSet.rows.emplace_back(std::move(row));
+            }
+
+            Row verticeRow;
+            verticeRow.values.emplace_back("space");
+            verticeRow.values.emplace_back("vertices");
+            verticeRow.values.emplace_back(statisItem.space_vertices);
```

Comment:
`statisItem.space_vertices` and `statisItem.space_edges` are the total count, right? Maybe we could use all capital letters, or add a blank row before.

Reply:
Yeah, use `Space `and `vertices `   to represent all the vertices under the space, and `Space `and `edges `to represent all the edges under the space.

Reply:
display as follows：
![image](https://user-images.githubusercontent.com/59197347/99926030-2c285480-2d7b-11eb-9a4f-fb0a413feaf2.png)